src/components/pokemonSelectorModal.html

```diff
@@ -61,7 +61,10 @@ <h5 class="modal-title text-light">Select a Pokémon</h5>&nbsp;
           <div class="form-row m-0" id="protein-filter">
             <div class="form-group col-md-6 col-6 p-0">
                 <label for="protein-search">Search</label>
-                <input id="protein-search" autocomplete="off" class="form-control" oninput="ProteinFilters.search.value(new RegExp(`(${this.value})`, 'i'))" placeholder="search here"/>
+                <input id="protein-search" autocomplete="off" class="form-control" placeholder="search here"
+                    oninput="ProteinFilters.search.value(this.value)"
+                    data-bind="value: ProteinFilters.search.value()"
+                />
```

Comment:
I'm pretty sure you can do this with the `textInput` binding
```suggestion
                <input id="protein-search" autocomplete="off" class="form-control" placeholder="search here"
                    data-bind="textInput: ProteinFilters.search.value()"
                />
```

Reply:
I committed your suggestion, and it still works! And it looks cleaner :)